openassessment/xblock/static/js/src/lms/oa_base.js

```diff
@@ -66,11 +72,33 @@ OpenAssessment.BaseView.prototype = {
      * @param {element} parentElement JQuery selector for the container element.
      */
     setUpCollapseExpand: function(parentElement) {
-        parentElement.on('click', '.ui-toggle-visibility__control', function(eventData) {
-                var sel = $(eventData.target).closest('.ui-toggle-visibility');
-                sel.toggleClass('is--collapsed');
-            }
-        );
+        var view = this;
+
+        $('.' + view.SLIDABLE_CONTROLS_CLASS, parentElement).each(function() {
+            $(this).on("click", function(event) {
+                event.preventDefault();
+
+                var $slidableControl = $(event.target).closest('.' + view.SLIDABLE_CONTROLS_CLASS);
+
+                var $container = $slidableControl.closest('li');
```

Comment:
In the old code, the encompassing container had a class of "ui-toggle__visibility". In the new code, there isn't a class for the encompassing container, just for the button, the overall control, and the content. What's the difference between having a specific ui-slidable__control vs. a ui-slidable? It seemed to me that ui-slidable should be the new ui-toggle__visibility, ui-slidable__control for the button, and ui-slidable__content for the content, but I think you had it a bit differently in your code for the staff area.

Maybe it makes sense to make ui-slidable__control the class for the button, and ui-slidable the class for the encompassing container?